Plugins/DataSink/MQTT/Readme.md

```diff
@@ -1,14 +1,14 @@
 ## MQTT
 
-The MQTT plugin publishes via MQTT the measurement values.
+The MQTT plugin publishes the measurement values via MQTT.
 
 Return to [main](./../Readme.md).
 
 ## MQTTConfig.config
 
 For the MQTT plugin the following settings are available:
 
-* <code>BrokerIP</code> - The IP of the system where the MQTT broker is running. The default is: <code>127.0.0.1</code>
+* <code>BrokerIP</code> - The IP of the system where the MQTT broker is running. The default is: <code>localhost</code>

```

Comment:
`127.0.0.1` will be always available. You can modify the `localhost` entry in the host file to point to somewhere else.

Reply:
see the other comment for a reply